### PR TITLE
add posts about wallets

### PR DESCRIPTION
**Pull request by lyricat**

### CODE REVIEW

File: docs/wallets/fennec.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Fennec
+sidebar_position: 2
+---
+
+[Fennec](https://github.com/fox-one/fennec) is an open source browser extension that allows you to easily manage your own crypto assets wallet of Mixin Network.
```
Comment (YaruiP) [inline docs/wallets/fennec.md:6]: ... manage your own crypto assets **in the Mixin Network wallet.**

File: docs/wallets/fennec.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+You can install Fennec from Chrome web store or Firefox Add-ons:
+
+- [Chrome Store](https://chrome.google.com/webstore/detail/fennec/eincngenkhohbbfpkohipekcmnkfamjp)
+- [Firefox Add-ons](https://addons.mozilla.org/en-US/firefox/addon/fennec/)
+
+Fennec uses Mixin Network's keystore files to identify your assets and manage your wallet. There are two ways to create your keystore files.
+
+By visiting [Mixin Developer Dashboard](https://developers.mixin.one/dashboard), creating an dApp and generating a keystore file.
```
Comment (YaruiP) [inline docs/wallets/fennec.md:15]: Change to : 

The first option is to create a  dApp from [Mixin Developer Dashboard](https://developers.mixin.one/dashboard). After creating the dApp, you can generate a keystore file.

File: docs/wallets/fennec.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+:::info
+**Mixin Developer Dashboard** is a service hosted by Mixin Core Team. You can use it to create two free dApps, but you will need to pay a fee for more dApps.
+:::
+
+By using the [Account Service providers](https://github.com/fox-one/fennec-asp), which generates a keystore file for you.
```
Comment (YaruiP) [inline docs/wallets/fennec.md:21]: The second option is to use the [Account Service providers](https://github.com/fox-one/fennec-asp) to generate a keystore file.

File: docs/wallets/fennec.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+:::info
+**Account Service Provider** is also an open source program that creating valid keystore files for you.
```
Comment (YaruiP) [inline docs/wallets/fennec.md:24]: 1. program that **creates** ...

2. 语义感觉有重复，上面已经表达了 generate keystore files. 这儿是想强调是open source的么？

File: docs/wallets/mixin-messenger.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+sidebar_position: 1
+---
+
+[Mixin Messenger](https://mixin.one/messenger) is an open source cryptocurrency wallet, which supports almost all popular cryptocurrencies.
+
+Mixin Messenger is developed by Mixin Core Team, who release it under the GPL-3.0 license.
+
+For users, please download the app from its official website [mixin.one](https://mixin.one/messenger).
+
+For developers, please check out its source code at here:
```
Comment (YaruiP) [inline docs/wallets/mixin-messenger.md:12]: **here at:**

**Review by YaruiP — COMMENTED** [review]
a few fixes

File: docs/wallets/fennec.md (hunk 4 above)
Comment (lyricat) [reply docs/wallets/fennec.md:24]: 嗯 是的

Comment (YaruiP) [reply docs/wallets/fennec.md:24]: 那要不info部分直接表达为：

**Account Service Provider** is an open source program

**Pull request merged**